### PR TITLE
Update forms.py

### PR DESCRIPTION
**Pull request by congyingTech**
Regexp has three parameter---regex, flags=0, message=None, message should be a type of string.

### CODE REVIEW

**Comment by congyingTech** [comment]
Regexp has three parameter---regex, flags=0, message=None, message should be a type of string.

**Comment by miguelgrinberg** [comment]
@congyingTech What are you trying to fix with this change? You just combined a long string split in two lines, there is no difference.

**Comment by congyingTech** [comment]
'Usernames must have only letters, numbers, dots or '
              'underscores')])
before my changed , I think two more quotes in your code，and when I run your code on server, it return an error, so I changed it like below:
             
       'Usernames must have only letters, numbers, dots or underscores'

**Comment by miguelgrinberg** [comment]
@congyingTech the code is correct, as far as I can see. Can you show me the error that you got?

**Comment by ScottMcCormack** [comment]
I think @congyingTech is getting with the extra string statement. Note that there is no comma between the two strings so there are only three parameters to `Regexp` and not four

```
 Regexp('^[A-Za-z][A-Za-z0-9_.]*$', 0,
               'Usernames must have only letters, numbers, dots or '
               'underscores')])
```

Also FYI the following code is valid python:

```
>>> example = "Some example"" text"
>>> example
'Some example text'
```

File: app/auth/forms.py [hunk 1]
```diff
@@ -19,8 +19,7 @@ class RegistrationForm(FlaskForm):
     username = StringField('Username', validators=[
         DataRequired(), Length(1, 64),
         Regexp('^[A-Za-z][A-Za-z0-9_.]*$', 0,
-               'Usernames must have only letters, numbers, dots or '
```
Comment (ScottMcCormack) [inline app/auth/forms.py]: Note there is no `,` after this line so it is not an additional parameters

**Pull request closed without merging**